tests/resources/test_texture.py

```diff
@@ -109,3 +109,14 @@ def test_unsupported_shapes():
         a = np.zeros((10, 10, 10, i), np.float32)
         with pytest.raises(ValueError):
             gfx.Texture(a, dim=3)
+
+
+def test_contiguous():
+    im0 = np.zeros((100, 100), np.float32)
+    im1 = im0[10:-10, 10:-10]
```

Comment:
I think that non-contiguous textures work because I do this all the time, unless we're doing an additional step in fastplotlib which makes it contiguous. I can check later.